tools/languagelocalizer/languageLocalizer.js

```diff
@@ -11,8 +11,7 @@ import inquirer from 'inquirer';
 import fetch from 'node-fetch';
 import WBK from 'wikibase-sdk';
 
-const I18N_SUBMODULE_PATH = '../../src/apps/vpn/translations/i18n';
-const DEFAULT_MOZILLAVPN = '../../build/src/mozillavpn';
+const I18N_SUBMODULE_PATH = '../src/apps/vpn/translations/i18n';
```

Comment:
I think this need's to be `../src/translations/i18n` as the app's folder was flattended